invenio_cli/helpers/cli_config.py

```diff
@@ -113,9 +113,9 @@ def get_db_type(self):
         """Returns the database type (mysql, postgresql)."""
         return self.config[CLIConfig.COOKIECUTTER_SECTION]["database"]
 
-    def get_es_version(self):
-        """Returns the database type (mysql, postgresql)."""
-        return self.config[CLIConfig.COOKIECUTTER_SECTION]["elasticsearch"]
+    def get_search_version(self):
```

Comment:
very minor: what about `get_search_type`, just to be consistent with the DB, since now is not just version but also a "different" technology.